### PR TITLE
Add linux-static-x64 and linux-static-arm64 images

### PR DESCRIPTION
**Pull request by aqrln**
Add linux-static-x64 and linux-static-arm64 images

Ref: https://github.com/prisma/team-orm/issues/115

### CODE REVIEW

**Comment by Jolg42** [comment]
@aqrln something is failing in the build about debian stretch

**Review by Jolg42 — APPROVED** [review]

File: .buildkite/pipeline.sh [hunk 1]
```diff
@@ -7,10 +7,14 @@ if [[ "$BUILDKITE_BRANCH" == "main" ]]; then
       command: cd rhel && make build && make push
       branches: main
 
-    - label: \":linux: Musl build images\"
+    - label: \":linux: Musl (dynamic) build images\"
       command: cd musl && make build && make push
       branches: main
 
+    - label: \":linux: Musl (static) build images\"
```
Comment (janpio) [inline .buildkite/pipeline.sh:14]:
```suggestion
    - label: \":linux: Linux/Musl (static) build images\"
```

**Review by janpio — APPROVED** [review]
Optional: Suggested to add `Linux/` to the job labels to make it easier to connect to what it builds (= `linux-static`)

**Comment by aqrln** [comment]
@janpio

> Suggested to add Linux/ to the job labels to make it easier to connect to what it builds (= linux-static)

Actually it looks like the directory (`linux-static`) is specified explicitly after the label anyway:

image

I think it's fine to leave it out to avoid renaming other jobs in this PR for consistency.

**Comment by steebchen** [comment]
Thanks :D

**Pull request merged**